frontend/README.md

```diff
@@ -1,20 +1,15 @@
 Tobira Frontend
```

Comment:
This is basically just a rehash of what's in the main README already. Do we really want this? :thinking:

Reply:
The reason for me keeping this file was that I expect this to evolve in which case it should be reasonable complete when it comes to the important information.

Reply:
I speculated as much. Still wanted to have mentioned it, I guess. ^_^